docs/src/pages/components/pagination/PaginationLink.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { MemoryRouter as Router } from 'react-router';
+import { Link as RouterLink } from 'react-router-dom';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+import Pagination from '@material-ui/lab/Pagination';
+import PaginationItem from '@material-ui/lab/PaginationItem';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      '& > *': {
+        marginTop: theme.spacing(2),
+      },
+    },
+  }),
+);
+
+// The use of React.forwardRef will no longer be required for react-router-dom v6.
+// See https://github.com/ReactTraining/react-router/issues/6056
```

Comment:
You sure? Seems more like the comment was just removed from similar demos rather than it no longer being required, for example:
https://material-ui.com/guides/composition/#button

Reply:
I think so. The demo you have linked to uses the ref prop, not innerRef.

Reply:
The comment was not completely accurate. Forward ref can be removed when the only use case is to forward the ref. In the button demo, we apply a to prop.

Reply:
ButtonBase component prop expects a component that holds a ref.

Reply:
Which RouterLink can holds out of the box.

Reply:
Ah, gotcha, the intermediate function is unnecessary.

Reply:
:)